test/asserts/big-number-assert_test.js

```diff
@@ -20,31 +20,71 @@ const Assert = BaseAssert.extend({
  */
 
 describe('BigNumberAssert', () => {
-  it('should throw an error if the input value is not a big number', () => {
-    const choices = [[], {}, ''];
+  [undefined, { validateSignificantDigits: true }, { validateSignificantDigits: false }].forEach(option => {
+    describe(`with option '${option ? `{ validateSignificantDigits: ${option.validateSignificantDigits} }` : undefined }'`, () => {
+      it('should throw an error if the input value is not a big number', () => {
+        const choices = [[], {}, ''];
 
-    choices.forEach(choice => {
-      try {
-        new Assert().BigNumber().validate(choice);
+        choices.forEach(choice => {
+          try {
+            new Assert().BigNumber(option).validate(choice);
 
-        should.fail();
-      } catch (e) {
-        e.should.be.instanceOf(Violation);
-      }
-    });
-  });
+            should.fail();
+          } catch (e) {
+            e.should.be.instanceOf(Violation);
+          }
+        });
+      });
 
-  it('should expose `assert` equal to `BigNumber`', () => {
-    try {
-      new Assert().BigNumber().validate();
+      it('should expose `assert` equal to `BigNumber`', () => {
+        try {
+          new Assert().BigNumber(option).validate();
 
-      should.fail();
-    } catch (e) {
-      e.show().assert.should.equal('BigNumber');
-    }
-  });
+          should.fail();
+        } catch (e) {
+          e.show().assert.should.equal('BigNumber');
+        }
+      });
+
+      it('should accept a big number', () => {
+        const choices = [
```

Comment:
Inline in the `forEach`?

Reply:
It was already like this and it's consistent across the codebase (34 hits).

I don't think it's worth the trouble and heavily pollutes the diff.